libs/core/src/components/sage-checkbox/sage-checkbox.tsx

```diff
@@ -0,0 +1,117 @@
+import { Component, h, Prop, Host, Event, EventEmitter } from '@stencil/core';
+
+@Component({
+  tag: 'sage-checkbox',
+  styleUrl: 'sage-checkbox.scss',
+  shadow: true,
+})
+export class SageCheckbox {
+  /**
+   * Whether or not the checkbox is checked.
```

Comment:
Can we update the wording here to be more complete? e.g Indicates if the checkbox is checked or not. Just an example here